### ISSUE TITLE
Custom network requires port number

### ISSUE THREAD

**Issue by aion-kimcodeashian** [blockxlabs/aiwa#15]
Before filing a new issue, please provide the following information.

#### AIWA version
1.1

#### Installed from Chrome Web Store or Github release?
Chrome Webstore

#### Chrome version?
Version 72.0.3626.96

#### OS and version?
Windows 10

#### Which provider are you using?
Custom - Nodesmith URL

#### Expected vs actual behaviour
Requires URL & Port number, Nodesmith does not need a Port number. As a result, unable to connect via Nodesmith URL alone.

**Comment by kushptlapigarage** [comment]
Already reported by Brendan from Node Smith and new version is on it's way

**Comment by kushptlapigarage** [comment]
If anybody is facing this issue and port number is not provided for http/https
* For HTTP endpoint enter port number as 80
* For HTTPS endpoint enter port number as 443

The feature was mostly developed with Fund Owners and Developers connecting local node which normally does not have Query Parameter or URLs with paths.

Ff anyone is blocked by this they can create a small proxy  'Local' <---> 'Remote'. I tried with NodeJS Express framework and worked fine for me at least Aiwa's major features.

**Comment by kushptlapigarage** [comment]
@aion-kimcodeashian could you please test and let me know.

**Comment by aion-kimcodeashian** [comment]
Hey @kushptlapigarage - seems to be working which is great! 

If there could be an info button or details available for the custom URL option - that will remind users about the correct port number for HTTP/HTTPS connections, I think that would be super helpful 👍

**Comment by kushptlapigarage** [comment]
Closing the issue
